lib/main_common.pm

```diff
@@ -735,6 +731,10 @@ sub load_inst_tests {
         if (get_var("LIVECD") && !is_krypton_argon) {
             loadtest "installation/livecd_network_settings";
         }
+        # Run system_role/desktop selection tests if using the new openSUSE installation flow
+        if (is_opensuse && get_var("SYSTEM_ROLE_FIRST_FLOW")) {
```

Comment:
All opensuse, not just tw & leap15 & kubic? - or you want to keep the variable?

Reply:
Leap also has the SYSTEM_ROLE_FIRST_FLOW

This is different from the Kubic install ordering changes I'm also doing - this the change that's already live in Leap and Tumbleweed - but the original ordering of that new flow doesn't work for the LiveCD installer - as you can see in test 644270

Reply:
We wont be removing the `SYSTEM_ROLE_FIRST_FLOW` variable, it's set by the main.pm for the appropriate product runs now